mrmustard/lab_dev/circuit_components_utils/b_to_ch.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2024 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+The class representing an operation that changes Bargmann into phase space.
+"""
+
+from __future__ import annotations
+
+from mrmustard.physics import triples
+
+from ..transformations.base import Map
+from ...physics.ansatz import PolyExpAnsatz
+from ...physics.wires import ReprEnum
+from ..utils import make_parameter
+
+__all__ = ["BtoCH"]
+
+
+class BtoCH(Map):
+    r"""
+    The `s`-parametrized ``Dgate`` as a ``Map``.
+
+    Used internally as a ``Channel`` for transformations between representations.
```

Comment:
```suggestion
    This is an unphysical component whose purpose is to modify the internal representation of another component. In particular it transforms between the Bargmann representation and the s-parametrized Characteristic functions. Note that it can be applied to a subset of modes.
```